components/compliance-service/api/reporting/server/server.go

```diff
@@ -285,7 +285,13 @@ func (srv *Server) ListNodes(ctx context.Context, in *reporting.Query) (*reporti
 
 // ReadNode returns a node based on id
 func (srv *Server) ReadNode(ctx context.Context, in *reporting.Id) (*reporting.Node, error) {
-	node, err := srv.es.GetNode(in.Id)
+	formattedFilters := formatFilters([]*reporting.ListFilter{})
```

Comment:
Why call `formatFilters` here and not create an empty `map[string][]string`?

Reply:
I considered that, but I thought it was worth having all of these go through the same code path in case we wanted to change how they all work at once (add some default filter, change the format, etc).